vars/pythonSetupPyPipeline.groovy

```diff
@@ -337,3 +361,12 @@ def deployDockerImage(dockerRegistryUrl, dockerRegistryCredentialsId, dockerFile
     }
   }
 }
+
+def isDeployDocsPluginInstalled() {
+  if (sh(script: "pip list | grep -F ghp-import", returnStatus: true) == 0) {
```

Comment:
If you would use `pip show ghp-import` you would not need to use grep and instead rely directly on the exit code (0 if available or 1 if not).

I think this should work as well:

```
return sh(script: "pip show ghp-import", returnStatus: true) == 0
```

Reply:
@martinm82 , I applied your suggestion.